modules/gsp-cluster/external-dns.tf

```diff
@@ -7,26 +7,36 @@ data "aws_iam_policy_document" "external_dns" {
     ]
 
     resources = [
-      "arn:aws:route53:::hostedzone/${var.cluster_domain_id}"
+      "arn:aws:route53:::hostedzone/${var.cluster_domain_id}",
     ]
   }
 
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "route53:ChangeResourceRecordSets",
+    ]
+
+    resources = "${formatlist("arn:aws:route53:::hostedzone/%s", aws_route53_zone.subdomain.*.zone_id)}"
+  }
```

Comment:
Based on the comments in the documentation, you probably want to duplicate this in the `cert_manager.tf` as well.